extensions/capability-negotiation.md

```diff
@@ -176,6 +184,18 @@ The server MUST accept the `CAP` command at any time, including after registrati
 The LS subcommand is used to list the capabilities supported by the server.  The client
 should send an LS subcommand with no other arguments to solicit a list of all capabilities.
 
+The format of a server-sent `CAP LS` message is:
+
+    CAP LS <version>
+
+The format of a server-sent `CAP LS` message is:
+
+    CAP <nick> LS :[<extension 1> ... [<extension n>]]
```

Comment:
Do we want to allow empty strings here? I suppose nothing is forbidding this in the spec…

Reply:
Spec explicitly allows empty strings for `LS` and `LIST`

Reply:
ACK!

Reply:
Actually, this makes me wonder about `ACK` and `NAK`, should they support it?

Reply:
`NAK` uses the phrase "space-seperated list of capabilities", which is the same as in `LS` and `LIST`.

Reply:
Should I add language to explicitly permit empty `REQ`, `ACK`, and `NAK`?

Reply:
I don't see any use-case for allowing `REQ` with an empty string.

Do servers in the wild support it?

These three commands need to all accept it or all reject it, since a server will reply to a `REQ` with an `ACK`/`NAK` with the same parameter.